ShipGame/.vs/ShipGameWindows/xs/UserPrefs.xml

```diff
@@ -0,0 +1,27 @@
+﻿<Properties>
```

Comment:
@SimonDarksideJ I didn't realise this .vs sub-dir exists. I'll delete this and do another push, so you may have to do another review :).